docs/reference/cat/tasks.asciidoc

```diff
@@ -0,0 +1,113 @@
+[[cat-tasks]]
+=== cat task management API
+++++
+<titleabbrev>cat task management</titleabbrev>
+++++
+
+beta::[The cat task management API is new and should still be considered a beta feature. The API may change in ways that are not backwards compatible.]
```

Comment:
I am not totally sure why task management is still beta. Perhaps, it is time to start a discussion about removal of this label. But since the non-cat API is marked beta, it is appropriate to use this label here as well.

Reply:
I've opened #47378 to discuss this more.